eng/Version.Details.xml

```diff
@@ -265,11 +265,11 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>0f64b267ac0552f07b06f18103581b880c8c53c6</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Runtime.MonoAOTCompiler.Task" Version="6.0.0-preview.3.21167.1">
+    <Dependency Name="Microsoft.NET.Runtime.MonoAOTCompiler.Task" Version="6.0.0-preview.3.21174.2">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>0f64b267ac0552f07b06f18103581b880c8c53c6</Sha>
```

Comment:
Recommend aligning the SHAs w/ #31255 to avoid conflicts between this and that. But, I agree it'll be better all 'round if that goes in first (with new versions from dotnet/runtime).